language/bytecode-verifier/transactional-tests/tests/locals_safety/while_move_local_2.mvir

```diff
@@ -5,14 +5,17 @@ main() {
     let b: bool;
     x = 0;
     b = true;
-    while (true) {
-        if (copy(b)) {
-            y = move(x);
-        } else {
-            // invalid move
-            x = move(y);
-        }
-        b = false;
-    }
-    return;
+
+label loop:
```

Comment:
This test changed a bit more than I would have liked, the CFG here and the one for `while (true)` are very different, as `while (true)` is not guaranteed to enter the loop, as far as the bytecode verifier is concerned